jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,356 @@
+```
+
+Alternatively, you can use the API token as the username for HTTP Basic Authentication, by passing the `-u` flag to the `curl` command:
+
+```sh 
+curl -u <circle-token>: "https://circleci.com/api/..."
+```
+
+**Note:** The colon ":" notifies `curl` that there is no password being passed.
+
+## API Endpoints
+
+The CircleCI v2 API, and its associated endpoints allow you to make HTTP calls to designated endpoints developed in the underlying CircleCI API architecture. These endpoints provide programmatic access to CircleCI services, including pipelines, workflows, and jobs. 
+
+Before working with the API and making calls, you should first have an undestanding of the endpoints currently available. The table below lists the endpoints currently available. 
+
+Endpoint       | Description                       
+-----------|-------------------------------------------------------
+`GET /workflow/:id ` | This endpoint enables users to return an individual Workflow based on the `id` parameter being passed in the request
+`GET /workflow/:id/jobs` | This endoint enables users to retrieve all Jobs associated with a specific workflow, based on its unique `id`.
+`GET /project/:project_slug`  | This endpoint enables users to retrieve a specific Project by its unique slug.
+`POST /project/:project_slug/pipeline` | This endpoint enables users to retrieve an individual project by its unique slug.
+`GET /pipeline/:id` | This endpoint enables users to retrieve an individual pipeline, based on the `id` passed in the request.
+`GET /pipeline/:id/config`  | This endpoint enables users to retrieve the configuration of a specific pipeline.
+`GET /project/:project_slug/pipelines/[:filter]`  | This endpoint enables users to retrieve the most recent set of pipelines for a Project.
+
+## API Syntax
+
+When making an API request, make sure you follow standard REST API syntax and formatting. Adhering to proper REST API syntax ensures that the API server can properly process your request and return a valid JSON response. To make a request to the CircleCI API, use the following format:
+
+`https://circleci.com/api/v2`
+
+Where:
+
+- `https://circleci.com` is the resource URL for the API being called.
+- `api` is the class being called.
+- `v2` is the API version.
+
+# Getting Started with the API
+
+The CircleCI API v2 is backwards-compatible with previous API versions in the way it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as gh/CircleCI-Public/circleci-cli, which is a “triplet” of the project type, the name of your “organization”, and the name of the repository. For the project type you can use `  github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which is supported in API v2. The `organization` is your username or organization name in your version control system.
```

Comment:
```suggestion
The CircleCI API v2 is backwards-compatible with previous API versions in the way it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as gh/CircleCI-Public/circleci-cli, which is a “triplet” of the project type, the name of your “organization”, and the name of the repository. For the project type you can use ` github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which are supported in API v2. The `organization` is your username or organization name in your version control system.
```

Reply:
just a fix for the literal formatting that had got a little messed up due to a space - can't actually tell where that is looking at this change request though!